src/index.tsx

```diff
@@ -1,29 +1,26 @@
 import { ColorModeScript } from "@chakra-ui/react";
 import * as React from "react";
 import * as ReactDOM from "react-dom/client";
-import { App } from "./App";
+import WrappedApp from "./App";
 import reportWebVitals from "./reportWebVitals";
 import * as serviceWorker from "./serviceWorker";
 
-
 const container = document.getElementById("root");
 if (!container) throw new Error("Failed to find the root element");
 const root = ReactDOM.createRoot(container);
 
+// root.render(
+//   <React.StrictMode>
+//     <ColorModeScript />
+//     <WrappedApp />
+//   </React.StrictMode>,
+// );
 root.render(
-  <React.StrictMode>
+  <>
     <ColorModeScript />
-    <App />
-  </React.StrictMode>,
+    <WrappedApp />
+  </>,
 );
```

Comment:
why remove `React.StrcitMode` ?

Reply:
When I used `React.StrcitMode` I got this error 
<img width="1296" alt="Screenshot 2024-08-15 at 4 35 05 PM" src="https://github.com/user-attachments/assets/30e6899e-0c93-4bce-a2ce-67bd0162af58">
so, to solve it I remove that.